src/app/features/login/login.component.ts

```diff
@@ -1,11 +1,20 @@
-import { Component, ViewEncapsulation } from '@angular/core'; 
-
+import { Component, ViewEncapsulation } from '@angular/core';
+import { NgForm } from '@angular/forms';
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.css'],
   encapsulation: ViewEncapsulation.ShadowDom,
 })
 export class LoginComponent {
-    MyImageSvg  = 'assets/img/logo.svg';
-}
+  submitted=false;
+  formModel: FormModel = new FormModel();
+    LoginUser(formModel :FormModel) {
+      this.submitted = true;
+      console.log(formModel)
+    }
+ }
+ export class FormModel {
```

Comment:
You don't need to export this class
And I think this class is redundant here, as we can write form as plain object in that case